wsmaster/che-core-api-devfile/src/test/java/org/eclipse/che/api/devfile/server/validator/DevfileSchemaValidatorTest.java

```diff
@@ -53,7 +58,10 @@ public void shouldThrowExceptionOnValidationNonValidDevfile(
       schemaValidator.validateBySchema(getResource(resourceFilePath), false);
     } catch (DevfileFormatException e) {
       if (!Pattern.matches(expectedMessageRegexp, e.getMessage())) {
-        fail("DevfileFormatException with unexpected message is thrown: " + e.getMessage());
+        assertEquals(
+            e.getMessage(),
+            expectedMessageRegexp,
```

Comment:
We already have a check 
` if (!Pattern.matches(expectedMessageRegexp, e.getMessage())) {`
Why should we use `assertEquals` here? Just to print expected, actual?

Reply:
We need that because with just `fail` dev would have a message that error in different but would not have any clue what is the difference. With my changes this assert is not for real assertion but just to leverage built-in TestNG/IDE functionality to show what is the difference and do side-by-side compare. This was very handful for me while I was working on this PR.